src/index.ts

```diff
@@ -141,6 +146,9 @@ const moveTo = (a: Set<string>, b: Set<string>, key: string): string => {
   return key
 }
 
+const isExample = (filePath: string): boolean =>
+  filePath.split('/').some(name => name === 'examples') || filePath.split('\\').some(name => name === 'examples')
```

Comment:
could use https://nodejs.org/api/path.html#path_path_sep instead

Reply:
Thank you. It's a more elegant way.